src/js/components/AppListComponent.jsx

```diff
@@ -145,74 +145,69 @@ var AppListComponent = React.createClass({
     return (
       <table className={tableClassSet}>
         <colgroup>
-          <col style={{width: "28%"}} />
-          <col style={{width: "14%"}} />
-          <col style={{width: "14%"}} />
-          <col style={{width: "14%"}} />
-          <col style={{width: "14%"}} />
-          <col style={{width: "16%"}} />
+          <col className="appid" />
+          <col className="ram" />
+          <col className="cpu" />
+          <col className="status" />
+          <col className="tasks" />
+          <col className="health" />
+          <col className="actions" />
         </colgroup>
         <thead>
           <tr>
-            <th>
+            <th className="text-left appid">
```

Comment:
Thinking about it, we are actually moving away from the `app id` as we are going to show the basename of the app (the last part in the full path of the id) so it might make sense to do a quick refactoring and call this column `name` already.